### PR TITLE
feat: add ethereum as a default ethereum network.

### LINKED ISSUE #2345: Add default ethereum network to onboarding profile

### PR DESCRIPTION
**Pull request by Tuditi**
## Summary

closes: #2345

...

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by nicole-obrien** [comment]
👀

File: packages/desktop/views/dashboard/wallet/panes/account-summary/AccountSummaryPane.svelte [hunk 1]
```diff
@@ -5,22 +5,25 @@
     import { networks } from '@core/network'
 
     export let account: IAccountState
+
+    $: accountNetworks = $networks ?? []
```
Comment (MarkNerdi) [inline packages/desktop/views/dashboard/wallet/panes/account-summary/AccountSummaryPane.svelte]: networks is always at least an empty array, so we dont need this

File: packages/desktop/views/dashboard/wallet/panes/account-summary/AccountSummaryPane.svelte [hunk 2]
```diff
@@ -5,22 +5,25 @@
...
     "
 >
     <AccountSummary />
-    {#if $networks}
-        <AccountNetworkSummary {account} network={$networks[0]} />
-        {#if $networks[1]}
-            <AccountNetworkSummary {account} network={$networks[1]} />
-        {/if}
-    {/if}
+    {#each accountNetworks as network}
```
Comment (MarkNerdi) [inline packages/desktop/views/dashboard/wallet/panes/account-summary/AccountSummaryPane.svelte]: just use `$networks` here

File: packages/desktop/views/dashboard/wallet/tab-section/portfolio/components/NetworkTypePill.svelte [hunk 3]
```diff
@@ -23,6 +23,6 @@
             ? 'Ethereum'
             : chainType === EvmNetworkType.Isc
               ? 'IOTA Smart Contracts'
-              : undefined}
+              : 'Undefined Network'}
```
Comment (MarkNerdi) [inline packages/desktop/views/dashboard/wallet/tab-section/portfolio/components/NetworkTypePill.svelte]: localize

File: packages/shared/src/lib/core/network/stores/networks.store.ts [hunk 4]
```diff
@@ -11,15 +12,25 @@ export const networks: Writable<Network[]> = writable([])
...
             })
             .filter(Boolean) as IEvmNetwork[]
-        networks.set([stardustNetwork, ...chains])
+        _networks = [stardustNetwork, ...chains]
     }
+
+    if (features.network.evmNetworks.enabled) {
+        profile.evmNetworks?.forEach((evmNetwork) => {
+            const newNetwork = new EvmNetwork(evmNetwork)
+            _networks.push(newNetwork)
```
Comment (MarkNerdi) [inline packages/shared/src/lib/core/network/stores/networks.store.ts]: i would just make it one line:

```suggestion
            _networks.push(new EvmNetwork(evmNetwork)
)
```

File: packages/shared/src/lib/core/network/constants/default-l1-evm-network-configurations.constant.ts [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+export const DEFAULT_L1_EVM_NETWORK_CONFIGURATION: Readonly<{
+    [key in EvmNetworkId]: IPureEvmNetworkConfiguration
+}> = {
+    [SupportedNetworkId.Ethereum]: <IPureEvmNetworkConfiguration>{
+        name: 'Ethereum',
+        symbol: 'ETH',
+        ticker: 'ETH',
+        id: SupportedNetworkId.Ethereum,
+        chainId: ChainId.Ethereum,
+        coinType: DEFAULT_COIN_TYPE[SupportedNetworkId.Ethereum],
```
Comment (MarkNerdi) [inline packages/shared/src/lib/core/network/constants/default-l1-evm-network-configurations.constant.ts:17]: missing in `DEFAULT_COIN_TYPE`

File: packages/shared/src/lib/core/network/constants/default-l1-evm-network-configurations.constant.ts [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+        explorerUrl: DEFAULT_EXPLORER_URLS[SupportedNetworkId.Ethereum],
+        rpcEndpoint: 'https://ethereum-rpc.publicnode.com',
+    },
+    [SupportedNetworkId.Sepolia]: <IPureEvmNetworkConfiguration>{
+        name: 'Sepolia Testnet',
+        symbol: 'ETH',
+        ticker: 'ETH',
+        id: SupportedNetworkId.Sepolia,
+        chainId: ChainId.Sepolia,
+        coinType: DEFAULT_COIN_TYPE[SupportedNetworkId.Sepolia],
```
Comment (MarkNerdi) [inline packages/shared/src/lib/core/network/constants/default-l1-evm-network-configurations.constant.ts:27]: missing in `DEFAULT_COIN_TYPE`

**Review by MarkNerdi — APPROVED** [review]

**Pull request merged**